gamma_transients/lib/core.py

```diff
@@ -504,46 +716,64 @@ def triplet_hist2d_saver(table, bins, position_src=None):
     figure, ax0 = plt.subplots(1, 1, figsize=(5, 4))
 
     if position_src is not None:
+        ra_avg = np.average(table["RA"], axis=1) - position_src.ra.deg
 
-        ra_avg = np.average(table['RA'], axis=1) - position_src.ra.deg
+        dec_avg = np.average(table["DEC"], axis=1) - position_src.dec.deg
 
-        dec_avg = np.average(table['DEC'], axis=1) - position_src.dec.deg
+        ra_1, dec_1 = (
+            table["RA"][:, 0] - position_src.ra.deg,
+            table["DEC"][:, 0] - position_src.dec.deg,
+        )
 
-        ra_1, dec_1 = table['RA'][:, 0] - position_src.ra.deg, table['DEC'][:, 0] - position_src.dec.deg
-
-        ra_2, dec_2 = table['RA'][:, -1] - position_src.ra.deg, table['DEC'][:, -1] - position_src.dec.deg
+        ra_2, dec_2 = (
+            table["RA"][:, -1] - position_src.ra.deg,
+            table["DEC"][:, -1] - position_src.dec.deg,
+        )
 
         range_ra = np.max(
-            np.abs(np.array(np.max(table['RA']) - position_src.ra.deg, np.min(table['RA']) - position_src.ra.deg)))
+            np.abs(
+                np.array(
+                    np.max(table["RA"]) - position_src.ra.deg,
+                    np.min(table["RA"]) - position_src.ra.deg,
+                )
+            )
+        )
         range_dec = np.max(
-            np.abs(np.array(np.max(table['DEC']) - position_src.dec.deg, np.min(table['DEC']) - position_src.dec.deg)))
+            np.abs(
+                np.array(
+                    np.max(table["DEC"]) - position_src.dec.deg,
+                    np.min(table["DEC"]) - position_src.dec.deg,
+                )
+            )
+        )
         range_var = np.max([range_ra, range_dec])
         # rint(table['RA'].shape[1])
         range_ = [[-range_var, range_var], [-range_var, range_var]]
 
     else:
+        ra_avg = np.average(table["RA"], axis=1)
 
-        ra_avg = np.average(table['RA'], axis=1)
-
-        dec_avg = np.average(table['DEC'], axis=1)
+        dec_avg = np.average(table["DEC"], axis=1)
 
-        ra_1, dec_1 = table['RA'][:, 0], table['DEC'][:, 0]
+        ra_1, dec_1 = table["RA"][:, 0], table["DEC"][:, 0]
 
-        ra_2, dec_2 = table['RA'][:, -1], table['DEC'][:, -1]
+        ra_2, dec_2 = table["RA"][:, -1], table["DEC"][:, -1]
 
-        range_ra = np.max(np.abs(np.array(np.max(table['RA']), np.min(table['RA']))))
-        range_dec = np.max(np.abs(np.array(np.max(table['DEC']), np.min(table['DEC']))))
+        range_ra = np.max(np.abs(np.array(np.max(table["RA"]), np.min(table["RA"]))))
+        range_dec = np.max(np.abs(np.array(np.max(table["DEC"]), np.min(table["DEC"]))))
         range_var = np.max([range_ra, range_dec])
         print(range_var)
```

Comment:
remove verbose prints

Reply:
removed the print statement, but then all the variables range_ra, range_dec, range_var remain unused, also to remove or keep?

Reply:
if unused, then remove. If we need to print them, then just rollback this commit and add proper verbosity

Reply:
removed in its own commit